ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -1624,3 +1624,33 @@ def test_tensorflow_Cholesky(
         rtol=1e-4,
         atol=1e-4,
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        num_arrays=2,
+        shared_dtype=True,
+        min_value=0,
+        max_value=8,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.LeftShift"
+    ),
+)
+def test_tensorflow_RightShift(
```

Comment:
Seems like a typo. Shouldn't the function be named `test_tensorflow_LeftShift` ?